scripts/kubestellar

```diff
@@ -97,6 +86,24 @@ then
     exit 1
 fi
 
+function ensure_espw() {
+    if kubectl get Workspace "$espw_name" &> /dev/null; then
+	echo "espw workspace already exists -- using it:"
+	kubectl ws "$espw_name"
+    else 
+        kubectl ws create "$espw_name" --enter
+    fi
+    kubectl apply -f $SCRIPT_DIR/../config/exports $verbdir
```

Comment:
It looks great, thanks for making these changes. Just a question: do we need to apply the crds here to. For example: ``kubectl apply -f   $SCRIPT_DIR/../config/crds ``

Reply:
No.  It was always unneeded, and possibly a mistake, to put the CRDs into the ESPW.

Reply:
It install CRDs and APIExport in the ESPW was in our old instructions. For example see instruction to deploy the KubeStellar scheduler: https://docs.kubestellar.io/docs/coding-milestones/poc2023q1/kubestellar-scheduler/. Although the command was updated but the text still says: 

`` Create the Edge Service Provider Workspace (ESPW) and populate it with CRDs and APIs``

Reply:
We will need to revise our docs to remove any reference to that old instruction in the docs

Reply:
We have been making some mistakes for a long time.  We should stop making them.  Yes, we should update the doc too.

Reply:
@dumb0002 , regarding https://github.com/kcp-dev/edge-mc/pull/409/files#r1198971583 : those revisions should be made, but need not be coordinated with this PR.

Reply:
Sounds good.